api/pom.xml

```diff
@@ -31,9 +31,39 @@
             <archive>users@mvc-spec.java.net</archive>
         </mailingList>
     </mailingLists>
+
+    <properties>
+        <api_package>javax.mvc</api_package>
+        <spec_version>1.0</spec_version>
+        <spec_impl_version>1.0-SNAPSHOT</spec_impl_version>
```

Comment:
It looks like `${project.version}` currently is set to `1.0-SNAPSHOT`. So the spec-version-maven-plugin should be fine with this. So `${project.version}` and `${spec_impl_version}` have the same value currently. That's why I think we shouldn't hardcode the version.